.azure-devops/templates/trigger-tests.yml

```diff
@@ -92,15 +92,16 @@ jobs:
       serviceConnection: $(AzureServiceConnection)
 
 - job: 'testHub_job_1'
-  displayName: 'Test IoT Hub - certificate, config, core, and jobs'
+  timeoutInMinutes: 90
+  displayName: 'Test IoT Hub - certificate, config, core, jobs, state'
   condition: and(succeeded(), eq('${{ parameters.testHub }}', true))
   strategy:
     matrix: $[ variables['pythonVersions'] ]
     maxParallel: ${{ parameters.maxParallelHub }}
   steps:
   - template: run-tests-parallel.yml
     parameters:
-      path: 'azext_iot/tests/iothub/certificate azext_iot/tests/iothub/configurations azext_iot/tests/iothub/core azext_iot/tests/iothub/jobs'
+      path: 'azext_iot/tests/iothub/certificate azext_iot/tests/iothub/configurations azext_iot/tests/iothub/core azext_iot/tests/iothub/jobs azext_iot/tests/iothub/state'
```

Comment:
What's the avg time the state tests take now a days?

Reply:
latest run https://dev.azure.com/azureiotdevxp/aziotcli/_build/results?buildId=7301&view=logs&j=343657f0-47cf-572f-a49e-f5e39f674be5&t=b1e029c7-6d9b-5e3b-03b8-17684fd48aec

1263.60s setup    azext_iot/tests/iothub/state/test_hub_state_int.py::test_migrate_controlplane
870.57s teardown azext_iot/tests/iothub/state/test_hub_state_int.py::test_export_import_migrate_missing_hubs_error
389.90s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_export_import_dataplane
365.16s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_migrate_controlplane_with_create
350.51s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_export_import_controlplane_with_create
289.10s setup    azext_iot/tests/iothub/state/test_hub_state_int.py::test_mirgate_hub_dataplane_error
201.77s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_export_import_controlplane
174.70s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_migrate_dataplane
170.78s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_migrate_controlplane
59.44s setup    azext_iot/tests/iothub/state/test_hub_state_int.py::test_migrate_dataplane
33.15s setup    azext_iot/tests/iothub/state/test_hub_state_int.py::test_export_import_dataplane
26.23s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_export_import_migrate_missing_hubs_error
10.02s call     azext_iot/tests/iothub/state/test_hub_state_int.py::test_mirgate_hub_dataplane_error
the rest are 00.00s calls

overall the job took 1hr 15 min which is not good but not the worst

Reply:
also note that the tests _with_create also make a new hub (that I track and get rid of later)

Reply:
Ok, lets follow up to talk about optimizations in a future iteration.